jira.sgnl.yaml

```diff
@@ -383,6 +384,112 @@ entities:
+    description: Workspace entity in Jira
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: workspaceId
+        externalId: workspaceId
+        type: String
+        indexed: true
+        uniqueId: true
+
+  # WARNING: In order to sync Objects, you must specify the objectsQlQuery in the adapterConfig.
+  # otherwise Jira will return 400.
+  Object:
+    # https://developer.atlassian.com/cloud/assets/rest/api-group-object/#api-object-aql-post.
+    displayName: JiraObject
+    externalId: Object
+    description: Object entity in Jira
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: globalId
+        externalId: globalId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: workspaceId
+        externalId: workspaceId
+        type: String
+        indexed: true
+      - name: id
+        externalId: id
+        type: String
+      - name: label
+        externalId: label
+        type: String
+      - name: objectKey
+        externalId: objectKey
+        type: String
+      - name: objectType__workspaceId
+        externalId: objectType__workspaceId
+        type: String
+      - name: objectType__globalId
+        externalId: objectType__globalId
+        type: String
+      - name: objectType__id
+        externalId: objectType__id
+        type: String
+      - name: objectType__name
+        externalId: objectType__name
+        type: String
+      - name: objectType__description
+        externalId: objectType__description
+        type: String
+      - name: objectType__position
+        externalId: objectType__position
+        type: Int64
+      - name: objectType__created
+        externalId: objectType__created
+        type: DateTime
+      - name: objectType__updated
+        externalId: objectType__updated
+        type: DateTime
+      - name: objectType__objectCount
+        externalId: objectType__objectCount
+        type: Int64
+      - name: objectType__objectSchemaId
+        externalId: objectType__objectSchemaId
+        type: String
+      - name: objectType__inherited
+        externalId: objectType__inherited
+        type: Bool
+      - name: objectType__abstractObjectType
+        externalId: objectType__abstractObjectType
+        type: Bool
+      - name: objectType__parentObjectTypeInherited
+        externalId: objectType__parentObjectTypeInherited
+        type: Bool
+      - name: created
+        externalId: created
+        type: DateTime
+      - name: updated
+        externalId: updated
+        type: DateTime
+      - name: timestamp
+        externalId: timestamp
+        type: Int64
+      # TODO: We currently don't allow attributes to be start with a non-alphanumeric char.
```

Comment:
Please remove the attribute if it's not supported.

Reply:
Removed.